docs/infra/set-up-monitoring-notifications.md

```diff
@@ -0,0 +1,65 @@
+# Set up monitoring notifications
+
+## Overview 
+
+The Monitoring Module is responsible for presenting monitoring information from our application and infrastructure. It helps us identify performance bottlenecks, potential issues, and overall system health. With this module, we gain valuable insights to make data-driven decisions and ensure the reliability and availability of our services.
+
+## Features
+
+* Real time alerts: Module supports alerting rules to notify stakeholders when specific thresholds are breached.
+* Notification channels: it supports various notification channels, specifically email alerts notifications and integration with external Incident management tools like Splunk-On-Call or Pagerduty.
+
+## Requirements
+
+Before proceeding with setting up monitoring notifications, ensure that you have completed the setup for your application environment. Refer to the following guide for the necessary steps:
+
+[Set up application environment](./set-up-app-env.md)
+
+## Configure monitoring alerts
+
+The Monitoring Module is an integral part of the application environment setup, and any changes to it can be applied by running the following command to create or update application resources:
+
+```
+TF_CLI_ARGS_apply="-var=image_tag=<IMAGE_TAG>" make infra-update-app-service APP_NAME=app ENVIRONMENT=<ENVIRONMENT>
+```
```

Comment:
i think we can remove this section. instead, i suggest embedding this command rather than having it separate so that we make it explicit that we need to run this after the setup step (see my other comments)